src/api/custom-resolvers/mutations/account-sign-in.test.ts

```diff
@@ -93,26 +100,98 @@ describe('custom-resolvers/account-sign-in', () => {
     });
   });
 
-  describe('when the provided password is invalid', () => {
-    test('it should return success=false', async () => {
-      variables.password = `${mockPassword}-incorrect`;
+  describe('when account is not verified, but has a verificationHash and verificationExpiry has not expired', () => {
+    beforeEach(async () => {
+      jest.resetAllMocks();
+
+      sendConfirmEmailAddressEmailSpy = jest.fn(() => Promise.resolve(mockSendEmailResponse));
+      confirmEmailAddressEmail.send = sendConfirmEmailAddressEmailSpy;
+
+      await context.query.Account.updateOne({
+        where: { id: account.id },
+        data: {
+          isVerified: false,
+          verificationHash: mockAccount.verificationHash,
+        },
+      });
 
       result = await accountSignIn({}, variables, context);
+    });
 
-      const expected = { success: false };
+    test('it should reset the account`s verificationExpiry', async () => {
```

Comment:
would this be better being split into 2 test()?

Reply:
Great shout, have split into 2 tests and simplified the variables.